### PR TITLE
[MediaContent] Support ebook format

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Support ebook format

### API Changes ###

 - TCSACR-439

### CODE REVIEW

File: src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs [hunk 1]
```diff
@@ -471,6 +472,54 @@ private static List<MediaInfo> QueryMedia(SelectArguments arguments)
             }
         }
 
+        /// <summary>
+        /// Retrieves all matched ebook path with given <paramref name="keyword"/>.
```
Comment (wiertel) [inline src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs:476]: ebook path -> ebook paths, same in \<returns\>

File: src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs [hunk 2]
```diff
@@ -471,6 +472,54 @@ private static List<MediaInfo> QueryMedia(SelectArguments arguments)
...
+        /// </summary>
+        /// <privilege>http://tizen.org/privilege/mediastorage</privilege>
+        /// <privilege>http://tizen.org/privilege/externalstorage</privilege>
+        /// <param name="keyword"></param>
```
Comment (wiertel) [inline src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs:480]: description is empty, please add

File: src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/BookInfo.cs [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+        /// <summary>
+        /// Gets the author of book.
+        /// </summary>
+        /// <value>The author.</value>
+        /// <since_tizen> 9 </since_tizen>
+        public string Author { get; }
+
+        /// <summary>
+        /// Gets the published date as a formatted string.
```
Comment (wiertel) [inline src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/BookInfo.cs:64]: "published date, formatted as a string" would sound better,
you could also change "of book" to "of the book" in all properties here

File: src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs:476]: Done.

File: src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs:480]: Done.

File: src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/BookInfo.cs (hunk 3 above)
Comment (hsgwon) [reply src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/BookInfo.cs]: Done.

**Review by aferin — APPROVED** [review]

**Pull request merged**